iac-extensions/arm/src/main/java/org/sonar/iac/arm/plugin/ArmLanguage.java

```diff
@@ -19,19 +19,29 @@
  */
 package org.sonar.iac.arm.plugin;
 
+import java.util.Arrays;
+import org.sonar.api.config.Configuration;
 import org.sonar.api.resources.AbstractLanguage;
 
 public class ArmLanguage extends AbstractLanguage {
 
   public static final String KEY = "azureresourcemanager";
   public static final String NAME = "AzureResourceManager";
 
-  public ArmLanguage() {
+  private final Configuration configuration;
+
+  public ArmLanguage(Configuration configuration) {
     super(KEY, NAME);
+    this.configuration = configuration;
   }
 
   @Override
   public String[] getFileSuffixes() {
-    return new String[] {".bicep"};
+    String[] suffixes = Arrays.stream(configuration.getStringArray(ArmSettings.FILE_SUFFIXES_KEY))
+      .filter(s -> !s.trim().isEmpty()).toArray(String[]::new);
```

Comment:
```suggestion
      .filter(s -> !s.isBlank()).toArray(String[]::new);
```
I believe it would be the same?